test/C/src/DeadlineHandledAbove.lf

```diff
@@ -2,6 +2,17 @@
 // container reacts to that output.
 target C
 
+preamble {=
```

Comment:
Why do we need this preamble?

Reply:
It is because we use `lf_nanosleep`, which comes with `platform.h`, the implementations for which we want to compile as C